packages/now-build-utils/src/detect-builders.ts

```diff
@@ -175,6 +170,57 @@ async function checkConflictingFiles(
   return null;
 }
 
+function validateFunctions({ functions = {} }: Options) {
+  for (const [path, func] of Object.entries(functions)) {
+    if (path.length > 256) {
+      return {
+        code: 'invalid_function_glob',
+        message: 'Function globs must be less than 256 characters long.',
+      };
+    }
+
+    if (
+      func.maxDuration !== undefined &&
+      (func.maxDuration < 1 ||
+        func.maxDuration > 900 ||
+        !Number.isInteger(func.maxDuration))
+    ) {
+      return {
+        code: 'invalid_function_duration',
+        message: 'Functions must have a duration between 1 and 900.',
+      };
+    }
+
+    if (
+      func.memory !== undefined &&
+      ![...Array(50).keys()]
+        .slice(2, 48)
+        .map(i => i * 64)
+        .includes(func.memory)
```

Comment:
You don't need to allocate an array here. Instead, compare the [remainder](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder).

```js
if (128 <= memory && memory <= 3008 && memory % 64 === 0)
```

Reply:
🥇